services/manage/app/mixins/init-value.js

```diff
@@ -0,0 +1,25 @@
+import Mixin from '@ember/object/mixin';
```

Comment:
I realize this was copy/pasted, but can you rename this file to `init-value-mixin` to match the other mixin file names? You'll also have to change the `import` in your components